kDriveCore/Data/Models/Upload/UploadFile.swift

```diff
@@ -61,17 +68,22 @@ public class UploadFile: Object, UploadFilable {
     @Persisted public var creationDate: Date?
     @Persisted public var modificationDate: Date?
     @Persisted public var taskCreationDate: Date?
+    @Persisted public var progress: Double?
```

Comment:
At first glance this seems unnecessary, we maybe need to discuss it

Reply:
Discussed.